config.cli.js

```diff
@@ -33,6 +33,12 @@ const {argv} = require('yargs')
     describe: 'A unique secret, a random one will be generated if not provided',
     type: 'string',
   })
+  .option('httpauth', {
```

Comment:
This authentication handler is useful for other methods as well. For instance, a Single Sign On authentication system deployed via the web server can give `Authorization` to Flood server as well. 

I suggest that we make this more general. However, I haven't thought much about the details yet.

Reply:
we could change to `externalAuth` with a value specifying the type, and then according to the type in backend perform the relative logic